src/shared/bg_public.h

```diff
@@ -1127,10 +1127,24 @@ enum meansOfDeath_t
 // TODO: move other button definitions into gamelogic
 enum buttonNumber_t
 {
-  BUTTON_ATTACK3 = 9,
-  BUTTON_DECONSTRUCT = 13,
+	BTN_ATTACK       = 0,
+	// defined by daemon
+	BTN_TALK         = BUTTON_TALK, // = 1, // disables actions
```

Comment:
I find these confusing, how about changing the two lines to

```suggestion
	BTN_TALK         = BUTTON_TALK, // = 1 (redefined from daemon), disables actions
```